webapp/src/components/modals/update_run_status_modal.tsx

```diff
@@ -133,33 +153,53 @@ const UpdateRunStatusModal = ({
         }
     };
 
+    const description = () => {
+        const broadcastChannelCount = run?.broadcast_channel_ids.length ?? 0;
+        const followersChannelCount = runMetadata?.followers?.length ?? 0;
+
+        const OverviewLink = (...chunks: string[]) => (
+            <Link to={pluginUrl(`/runs/${playbookRunId}/overview`)}>
+                {chunks}
+            </Link>
+        );
+
+        if ((broadcastChannelCount + followersChannelCount) === 0) {
+            return formatMessage({
+                defaultMessage: 'This update will be saved to <OverviewLink>overview page</OverviewLink> and broadcasted to <OverviewLink>0 channels</OverviewLink>',
```

Comment:
I'd lean towards removing the final part of the sentence altogether if we are not going to broadcast. Thoughts, @abhijit-singh?
```suggestion
                defaultMessage: 'This update will be saved to <OverviewLink>overview page</OverviewLink>',
```